src/index.html

```diff
@@ -1,17 +1,245 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <meta
+        name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+        >
+        <meta 
+          http-equiv="X-UA-Compatible" 
+          content="ie=edge"
+        >
+        <title>HTML Form</title>
+        <link 
+          rel="stylesheet" 
+          href="./style.css"
+        >
```

Comment:
multi lines needs when tag have more then 2 attributes or length is more than 80
```suggestion
        <link rel="stylesheet" href="./style.css">
```